apps/docs/docs/api/javascript/create.mdx

```diff
@@ -33,3 +33,13 @@ const styles = stylex.create({
   }),
 });
 ```
+
+:::danger Requires Typescript 5+
+
+Since `stylex.create` uses the new `const` keyword in its type definition, it requires Typescript 5+.
```

Comment:
Just say that StyleX requires Typescript 5 or later. The rest of the context is unnecessary.

Reply:
I think requirements like version of TS or Node should be listed in the "installation" page rather than docs for specific APIs

Reply:
I tackled the comments. Should i resolve this conversation or should the one who opened it do it?